shell/common/shell_io_manager.h

```diff
@@ -24,9 +24,10 @@ class ShellIOManager final : public IOManager {
       GrBackend backend,
       sk_sp<const GrGLInterface> gl_interface);
 
-  ShellIOManager(sk_sp<GrDirectContext> resource_context,
-                 std::shared_ptr<fml::SyncSwitch> is_gpu_disabled_sync_switch,
-                 fml::RefPtr<fml::TaskRunner> unref_queue_task_runner);
+  ShellIOManager(
+      sk_sp<GrDirectContext> resource_context,
+      std::shared_ptr<const fml::SyncSwitch> is_gpu_disabled_sync_switch,
```

Comment:
nit: `const std::shared_ptr<const fml::SyncSwitch>&` to avoid an extra copy.

Reply:
Done